main.py

```diff
@@ -75,6 +78,13 @@ def export_streams_to_clipboard(self):
         clipboard.clear(mode=clipboard.Clipboard)
         clipboard.setText(text, mode=clipboard.Clipboard)
 
+    def import_streams_from_clipboard(self):
+        """Imports all streams from the users clipboard."""
+        streams = QtWidgets.QApplication.clipboard().text().split()
+
+        for stream in streams:
+            self.add_new_stream(stream)
```

Comment:
Yep, can't really think of any that wouldn't be catched by the try/catch in the `add_new_stream` though. Can you think of any?